mautic/mautic
# TPROD-439 Loading Quicksearch JS from NPM and loading the latest version

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [updating a dead JS library]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes https://mautic.atlassian.net/browse/TPROD-439



#### Description:

Changing from unmaintaned https://github.com/riklomas/quicksearch to https://github.com/DeuxHuitHuit/quicksearch

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Edit any report
3. In the Data tab there is a list of available columns and there is a search box on top. Make sure that by typing a column name the list of columns bellow is being filtered.
![Screenshot 2023-05-22 at 16 35 32](https://github.com/mautic/mautic/assets/1235442/4c2692ea-0e2e-4d0c-9037-432faa08781d)
